src/meta/windows_utils.rs

```diff
@@ -13,6 +13,32 @@ use super::{Owner, Permissions};
 
 const BUF_SIZE: u32 = 256;
 
+#[cfg(feature = "no-windows-permissions")]
+pub fn get_file_data(path: &Path) -> Result<(Owner, Permissions), io::Error> {
+    let owner = Owner::new("?".to_string(), "?".to_string());
```

Comment:
I think it would be better to make owner optional in the struct in case of windows(we could probably have https://github.com/Peltoche/lsd/pull/473 added in after that). 
Also have the user/group as `-` with a greyish colors simliar to when there are no permissions. Something like this.
![Screenshot 2021-02-13 at 8 45 14 PM](https://user-images.githubusercontent.com/14259816/107853505-8695f600-6e3c-11eb-96aa-fdaa37ea2eb2.png)

Reply:
That's a great idea -- I'll get on it today.  I may split the no-windows-perms stuff into a separate .rs file, so that it's easier to cfg out stuff to avoid warnings too.

Reply:
(sorry, life got in the way this past week. will get to it when I can!)